tools/webpack/configs/standard.mjs

```diff
@@ -14,57 +15,122 @@ import { BundleAnalyzerPlugin } from 'webpack-bundle-analyzer'
  * using --env foo=bar --env biz=baz
  */
 export default (env) => {
-  return merge(commonConfig(env), {
-    target: 'web',
-    entry: {
-      'nr-loader-rum': path.join(env.paths.src, 'cdn/lite.js'),
-      'nr-loader-rum.min': path.join(env.paths.src, 'cdn/lite.js'),
-      'nr-loader-full': path.join(env.paths.src, 'cdn/pro.js'),
-      'nr-loader-full.min': path.join(env.paths.src, 'cdn/pro.js'),
-      'nr-loader-spa': path.join(env.paths.src, 'cdn/spa.js'),
-      'nr-loader-spa.min': path.join(env.paths.src, 'cdn/spa.js'),
-      ...(env.SUBVERSION !== 'PROD' && { 'nr-loader-experimental': path.resolve(env.paths.src, 'cdn/experimental.js') }),
-      ...(env.SUBVERSION !== 'PROD' && { 'nr-loader-experimental.min': path.resolve(env.paths.src, 'cdn/experimental.js') })
+  const entryGroups = [
+    {
+      asyncChunkName: 'nr-rum',
+      entry: {
+        'nr-loader-rum': path.join(env.paths.src, 'cdn/lite.js'),
+        'nr-loader-rum.min': path.join(env.paths.src, 'cdn/lite.js')
+      },
+      plugins: [
+        new webpack.IgnorePlugin({
+          checkResource: (resource, context) => {
+            if (context.match(/features\/utils/) && resource.indexOf('aggregate') > -1) {
+              // Only allow page_view_event, page_view_timing, and metrics features
+              return !resource.match(/page_view_event\/aggregate|page_view_timing\/aggregate|metrics\/aggregate/)
```

Comment:
Did we determine if maintaining this list is actually necessary?  Or will it get tree-shaken out anyway?

Reply:
I did not try but I can.

Reply:
Yea we will have to maintain this list. Without this plugin, all the features get included in the async chunk for the lite and pro agent.